include/seqan3/alignment/matrix/detail/alignment_trace_matrix_proxy.hpp

```diff
@@ -29,15 +29,17 @@ namespace seqan3::detail
  * The `trace_type` must be either a seqan3::detail::trace_directions value or a seqan3::detail::simd_conceptvector
  * over seqan3::detail::trace_directions.
  */
-template <typename trace_type>
+template <typename coordinate_type, typename trace_type>
 struct alignment_trace_matrix_proxy
 {
-    static_assert(std::same_as<trace_type, trace_directions> || simd_concept<trace_type>,
+    static_assert(std::same_as<trace_type, trace_directions> || simd_concept<trace_type> ||
+                  decays_to_ignore_v<trace_type>,
```

Comment:
Adapt the assert message that it may also be of type std::ignore.
(:nail_care: Also this could fit into the line above)